Bottom: ae3ce3608e05010955f12b5fa24914167e01d668
Top:    a786516f46fc5d96333f3c85fe1e0ddd7b3402c0
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:31:43 -0500

Fix credential copy to only run on daedalus

Credentials for all hosts end up on daedalus, so limit that play.


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index f009bdb9..0727917d 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -112,7 +112,7 @@
     assemble_scripts: true
 
 - name: Cleanup tasks
-  hosts: all
+  hosts: daedalus
   remote_user: ansibleremote
   become: true
   become_user: root
@@ -120,14 +120,7 @@
   tasks:
   - name: Copy credentials to the /etc dir
     copy:
-      src: ./credentials/{{ inventory_hostname }}
-      dest: /etc/ansible-credentials/
-      owner: root
-      group: root
-      mode: 'u=rw,g=,o='
-  - name: Copy build system credentials to /etc dir
-    copy:
-      src: ./credentials/buildsystem.novatech-llc.com
+      src: "{{ playbook_dir }}/credentials/"
       dest: /etc/ansible-credentials/
       owner: root
       group: root
